### ISSUE TITLE
[bug]: toggle feature in a project is broken [self-hosted]

### ISSUE THREAD

**Issue by jeddors** [makeplane/plane#5784]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

When I click the toggle button to enable/disable a feature (e.g. Cycles, Modules, Views, Pages...) on a project, I get an error message:

```
Error!
Something went wrong while updating project feature.
Please try again.
```
 
e.g. clicking the toggle button for "Modules" will trigger the following API call:
```
Request URL: http://localhost/api/workspaces/process-engineering/projects/eb09e5a7-8157-448b-abe5-1ee0b084a487/
Request Method: PATCH
Request Payload: {"module_view":false}
```

And the response:
```
Status Code: 400 Bad Request
Response: {"error":"The required key does not exist."}
```

The log file for the API:
```
api-1  | ERROR 2024-10-09 20:49:49,827 exception_logger 157 140072644827960 'project_id'
api-1  | Server Error
api-1  | WARNING 2024-10-09 20:49:49,829 log 157 140072743394104 Bad Request: /api/workspaces/process-engineering/projects/eb09e5a7-8157-448b-abe5-1ee0b084a487/
api-1  | 172.18.0.14:44272 - "PATCH /api/workspaces/process-engineering/projects/eb09e5a7-8157-448b-abe5-1ee0b084a487/ HTTP/1.1" 400
```

### Steps to reproduce

1. Go to "Settings" for any project
2. Go to "Features"
3. try toggle on/off any of the listed features ("Cycles", "Modules", "Views" etc etc)

### Environment

Production

### Browser

Mozilla Firefox

### Variant

Self-hosted

### Version

v0.23.0

**Comment by pushya22** [comment]
Hi @jeddors, Thanks for bringing this to our attention. We’ve noticed that this occurs in some rare scenarios, and we are actively working on a fix. This will be included in our next release.

**Comment by pushya22** [comment]
@jeddors, this issue has been resolved in the v0.23.1 release. Please upgrade to the latest version, and let me know if you encounter any further issues.
